pkgs/os-specific/linux/firmware/fwupd/default.nix

```diff
@@ -165,12 +166,13 @@ let
       pango
       tpm2-tss
       efivar
```

Comment:
`cairo`, `freetype` and `fontconfig` might not be needed any more.

[`1.5.12...1.6.0`](https://github.com/fwupd/fwupd/compare/1.5.12...1.6.0#diff-30d8f6be6320feeacf686be94f48c70869b52630e01ea625f0f15adc0d57c3e4L349-L355)

Reply:
Thanks for spotting! Correct on all 5, of course.